app/file_opening.py

```diff
@@ -0,0 +1,22 @@
+import os
+import json
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def file_open(name_of_file: str) -> dict:
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    config_path = os.path.join(current_dir, name_of_file)
+    with open(config_path) as f:
+        person_data = json.load(f)
+        shops = [Shop(shop) for shop in person_data["shops"]]
+        customers = [
+            Customer(customer) for customer in person_data["customers"]
+        ]
+        fuel_price = person_data["FUEL_PRICE"]

```

Comment:
Consider adding error handling for missing keys in the `person_data` dictionary. This can prevent `KeyError` exceptions if any expected key is missing. Using `dict.get()` with default values or try-except blocks can be helpful.